index.html

```diff
@@ -16,29 +16,24 @@
     <div class="container">
       <header>
         <!-- Nav Section -->
-        <nav class="menu">
+        <nav class="navbar">
           <a href="#" class="logo">My Logo</a>
 
-          <ul>
-            <li>
-              <a href="#" class="active" onclick="markMenuItem(this)"
-                >Portfolio</a
-              >
-            </li>
-            <li><a href="#" onclick="markMenuItem(this)">About</a></li>
-            <li><a href="#" onclick="markMenuItem(this)">Contact</a></li>
+          <ul class="nav-menu">
+            <li class="nav-item"><a href="#info" class="nav-link">Portfolio</a></li>
+            <li class="nav-item"><a href="#abouts" class="nav-link">About</a></li>
+            <li class="nav-item"><a href="#contacts" class="nav-link">Contact</a></li>
```

Comment:
- [ ] As requested by the previous reviewer. From the project requirements, you should stick to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tt1vCN6lfPrkflxz-0), But as you can see there are some differences between the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tt1vCN6lfPrkflxz-0) and your design

| Your Design ( menu background is not blured) |Figma Design (menu background is blured) |
| :---:   | :---: | 
| ![Screenshot from 2022-12-19 20-24-16](https://user-images.githubusercontent.com/92303891/208515114-8fe9c202-d181-426a-8f80-6f407c9243b1.png)| ![image](https://user-images.githubusercontent.com/92303891/208515053-7a805c0d-9494-4828-8c6f-3ddbdedd623a.png) |